### DISCUSSION TITLE
Overview "From address" different than Headers From address.

### DISCUSSION THREAD

**Discussion by johnbritto1** [postalserver/postal#1682] [Help with using Postal] [answered]
Hi,

Can anyone explain why the Overview  "From address" is different than the actual message headers "From address".

Please have a look at the attached screenshot.
![overview](https://user-images.githubusercontent.com/5947168/139631100-150821f5-8e4e-4c8e-ac5e-ee6555a0e988.png)
![message header](https://user-images.githubusercontent.com/5947168/139631108-6a84dfb8-06e3-4827-819b-054e8ba02acc.png)

**Comment by willpower232** [comment, marked as answer]
Interesting, how are you constructing the message to send to Postal?

**Reply by johnbritto1** [reply]
Hi,

We found that it is our app that is constructing the wrong SMTP from address. 

Thank you.
